pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -26,6 +27,25 @@ func (ac *OSSAccessControlService) Init() error {
 
 	ac.registerUsageMetrics()
 
+	return ac.RegisterRegistrantsRoles()
+}
+
+func (ac *OSSAccessControlService) RegisterRegistrantsRoles() error {
+	services := registry.GetServices()
```

Comment:
This part (getting registrations from services) is the same for OSS and Enterprise. Can we share this code? For example, like `Evaluate()` function.

Reply:
Yes, the thought crossed my mind, your confirmation is all I needed. :smile:  I'll do the change.

Reply:
I had a similar thought. I'm going to do it, however this forces me to make the RegisterFixedRole function public but it's not thread safe. I'll put it in another interface than the AccessControl one. Tell me what you think of the last changes :smile: